client/app/iplb/sslCertificate/order/iplb-ssl-certificate-order.controller.js

```diff
@@ -67,8 +67,9 @@ class IpLoadBalancerSslCertificateOrderCtrl {
     }
 
     orderFreeCertificate () {
-        const fqdn = this.newSsl.fqdn.split(",");
-
+        const fqdn = this.newSsl.fqdn.split(",").map(function(item){
```

Comment:
Good fix!
Is your ESLint enabled in your IDE?
Use ES6 :
```.map(item => item.trim());```

Reply:
ESLint is not enabled. I'll do that thanks :)